content/en/observability_pipelines/guide/custom-metrics-governance.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Custom Metrics Governance with Observability Pipelines
+kind: guide
+aliases:
+  - /metrics/custom_metrics/guide/custom-metrics-governance-drop-metrics-missing-specific-tags/
+  - /integrations/observability_pipelines/guide/custom-metrics-governance-drop-metrics-missing-specific-tags/
+  - /observability_pipelines/guide/custom-metrics-governance-drop-metrics-missing-specific-tags
+further_reading:
+- link: "/observability_pipelines/setup/"
+  tag: "Documentation"
+  text: "Set up Observability Pipelines"
+---
+
+## Overview
+
+Custom metrics provide visibility into all facets of your business, from application performance, infrastructure health, to business KPIs. To govern your custom metric volumes, Datadog offers several tools for cost visibility and control after your metrics are ingested, such as [estimated real-time custom metric usage][1], [usage attribution][2], and [Metrics without Limits™][3].
+
+This guide walks you through how to use Observability Pipelines to govern and control your custom metrics before they are ingested. More specifically, how to do the following: 
+- [Drop custom metrics missing specific tags or the metric's namespace](#drop-metrics-missing-specific-tags-or-the-metrics-namespace)
+- [Remove custom metric tags](#remove-custom-metric-tags)
+- [Set up rules to prevent cardinality spikes](#prevent-cardinality-spikes)
+
+## Prerequisites
+
+This guide assumes that you have already set up Observability Pipelines, a tool for collecting, processing, and routing telemetry data. If you're not familiar with Observability Pipelines, see the [Setup][4] documentation and [Vector Remap Language][5] for more information.
+
+## Drop metrics missing specific tags or the metric's namespace
+
+### Problem
+
+It is useful to break down your overall metrics usage with the [Usage Attribution][6] feature, by adding tags like `team` or `application`. This lets you see how many custom metrics a particular team or application is generating. However, custom metrics might be submitted without those crucial tags. This makes it difficult to attribute the metrics back to the specific team, service, or application generating them. 
+
+### Solution
+
+To prevent that issue, use Observability Pipelines to drop metrics missing tags that are useful for Usage Attribution. This can be done before the metrics are added to your account's custom metrics usage. 
```

Comment:
```suggestion
To prevent that issue, use Observability Pipelines to drop metrics missing tags that are meaningful to you. This can be done prior to metric ingestion and therefore before metrics contribute to your account's custom metrics usage. 
```